config/locales/en.yml

```diff
@@ -1061,7 +1061,6 @@ en:
       cannot_have_goto_page_before_routing_page: The question route 1 skips to cannot be before question %{question_number} - edit or delete this route
       cannot_have_goto_page_before_routing_page_secondary_skip: The question the route for any other answer skips to cannot be before the question it skips from - edit or delete this route
       cannot_route_to_next_page: Route 1 is not skipping any questions - edit or delete this route
-      cannot_route_to_next_page_no_next_page: Route 1 cannot skip any questions because question X is at the end of the form - delete this route
```

Comment:
We appear to have lost this error message just one commit after it was created, is this intentional?

Reply:
No, I accidentally left this in doing a rebase shuffle, I'll edit it out of the commit